src/chapters/reverse-proxy.xml

```diff
@@ -45,6 +45,99 @@
+                or addresses of any and all reverse proxy servers that will be accessing this Tomcat
+                instance directly.  Often it is run on the same system that runs Tomcat, but in other
+                cases (for example, when running Docker), it may be on a different system/container and
+                may need to be set to the actual IP address of the reverse proxy system.  Only proxy
+                servers listed in the <parameter>internalProxies</parameter> or
+                <parameter>trustedProxies</parameter> parameters will be allowed to manipulate the
+                remote IP address information.  The other parameters in this configuration line allow
+                you to control which headers coming from the proxy server(s) are used for various
+                remote host information.  They are as follows:
+            </para>
+            <informaltable frame="all">
+                <tgroup cols="2">
+                    <colspec colname="c1" colnum="1" colwidth="1*"/>
+                    <colspec colname="c2" colnum="2" colwidth="3.55*"/>
+                    <thead>
+                        <row>
+                            <entry>Parameter name</entry>
+                            <entry>Description</entry>
+                        </row>
+                    </thead>
+                    <tbody>
+                        <row>
+                            <entry><parameter>remoteIpHeader</parameter></entry>
+                            <entry>
+                                <para>The header that is queried to learn the client IP address
+                                    of the client that originated the request.  The standard
+                                    value is <code>X-Forwarded-For</code>, but can
+                                    be configured to any header you like.  The IP address in this
+                                    header will be available to Java applications in the
+                                    <code>request.getRemoteAddr()</code> method.</para>
+                            </entry>
+                        </row>
+                        <row>
+                            <entry><parameter>remoteIpProxiesHeader</parameter></entry>
+                            <entry>
+                                <para>The header that is queried to learn the IP address of the
+                                    proxy server that forwarded the request.  The default value
+                                    is <code>X-Forwarded-By</code>, but can be configured to
+                                    any header that fits your environment.  This value will only
+                                    be allowed by the valve if the proxy used is listed in the
+                                    <parameter>trustedProxies</parameter> parameter.  Otherwise
+                                    this value will be null.</para>
```

Comment:
> ... Otherwise this value will be null.

This is unlikely to mean much to those that are not working with things at the Java source level.

Reply:
Reworded.